free-programming-books.md

```diff
@@ -687,7 +686,6 @@ Kerridge (PDF) (Just fill the fields with any values)
 * [Naked objects](http://downloads.nakedobjects.net/resources/Pawson%20thesis.pdf) - Richard Pawson (PDF)
 * [OAuth - The Big Picture](https://pages.apigee.com/oauth-big-picture-ebook.html)
 * [Object-Oriented Reengineering Patterns](http://scg.unibe.ch/download/oorp/) - S. Demeyer, S. Ducasse and O. Nierstrasz
-* [Practicing Domain-Driven Design](https://leanpub.com/Practicing-DDD) - Scott Millett  *(Leanpub account or valid email requested)*
```

Comment:
This book has a $0 minimum. Do you think the notation should be different? see #2692

Reply:
It is only part 1 of 4 and it's not a work in progress. We have already removed incomplete books that were not a work a progress.

Reply:
At 160 pages, it seems substantial enough to be a book on its own. 

I suppose the key question is whether people wanting to learn about Domain-Driven Design would find these 10 free chapters useful or not. Perhaps folks with expertise in this area can weigh in.